core/Piwik.php

```diff
@@ -563,7 +563,14 @@ public static function redirectToModule($newModule, $newAction = '', $parameters
      */
     public static function isValidEmailString($emailAddress)
     {
-        return (preg_match('/^[a-zA-Z0-9_.+-]+@[a-zA-Z0-9_.-]+\.[a-zA-Z]{2,}$/D', $emailAddress) > 0);
+        static $zendEmailValidator;
+        if(!$zendEmailValidator) {
+            $zendEmailValidator = new \Zend_Validate_EmailAddress(array(
+                'mx' => false,
+                'deep' => false
+            ));
+        }
```

Comment:
It was introduced as `@deprecated` because Piwik's dependencies are too hard-coded. It's taken us ~5 months or so to try to remove it, and we are still one month away from doing it. You can use it for now, though you should try to avoid where possible.